src/Reveal.Sdk.Dom.Tests/Visualizations/GridVisualizationFixture.cs

```diff
@@ -0,0 +1,371 @@
+﻿using Newtonsoft.Json.Linq;
+using Reveal.Sdk.Dom.Core.Serialization;
+using Reveal.Sdk.Dom.Data;
+using Reveal.Sdk.Dom.Filters;
+using Reveal.Sdk.Dom.Visualizations;
+using Reveal.Sdk.Dom.Visualizations.Settings;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Visualizations
+{
+
+    public class GridVisualizationTests
+    {
+        [Fact]
+        public void GridVisualization_DefaultConstructor_ShouldInitializeWithDefaultValues()
+        {
+            // Act
+            var gridVisualization = new GridVisualization();
+
+            // Assert
+            Assert.NotNull(gridVisualization);
+            Assert.Equal(ChartType.Grid, gridVisualization.ChartType);
+            Assert.NotNull(gridVisualization.Columns);
+            Assert.Empty(gridVisualization.Columns);
+        }
+
+        [Theory]
+        [InlineData("TestTitle", null)]
+        [InlineData(null, null)]
+        public void GridVisualization_ConstructorWithTitleAndDataSource_ShouldSetValues(string title, DataSourceItem dataSourceItem)
+        {
+            // Act
+            var gridVisualization = new GridVisualization(title, dataSourceItem);
+
+            // Assert
+            Assert.Equal(title, gridVisualization.Title);
+            Assert.Equal(ChartType.Grid, gridVisualization.ChartType);
+            Assert.NotNull(gridVisualization.Columns);
+            Assert.Empty(gridVisualization.Columns);
+        }
+
+        [Theory]
+        [InlineData(true, 100)]
+        [InlineData(false, 50)]
+        public void GridVisualizationSettings_PageSettings_ShouldSetCorrectValues(bool pagingEnabled, int pageSize)
```

Comment:
This should be moved to another new fixture: `GridVisualizationSettingsFixture`

Reply:
Also, check if we need Unit test for those types of fields with simple getter and setter

Reply:
Oke I will create another test classes for those settings with another issue